index.js

```diff
@@ -19,9 +19,34 @@ mongoose
     // Before adding any recipes to the database, let's remove all existing ones
     return Recipe.deleteMany()
   })
+  .then(() => Recipe.syncIndexes())
   .then(() => {
-    // Run your code here, after you have insured that the connection was made
+
+    return Recipe
+      .create([
+        {
+          title: 'Tortilla de patata',
+          level: 'Easy Peasy',
+          ingredients: ['patata', 'huevo', 'sal'],
+          cuisine: 'traditional',
+          dishType: 'main_course',
+          duration: 20,
+          creator: 'yo misma',
+          created: ''
+        }
+      ])
+  })
```

Comment:
- Cuidado con lo que le pasas al `create`, aquí le estás pasando un `array` con un objeto, y tienes que pasarle únicamente un objeto.

```suggestion
return Recipe
      .create(
        {
          title: 'Tortilla de patata',
          level: 'Easy Peasy',
          ingredients: ['patata', 'huevo', 'sal'],
          cuisine: 'traditional',
          dishType: 'main_course',
          duration: 20,
          creator: 'yo misma',
          created: ''
        }
      )
```